### PR TITLE
[4.1] WebAuthn hints

### PR DESCRIPTION
**Pull request by kitepascal**
Pull Request for Issue # .

### CODE REVIEW

File: administrator/language/en-GB/plg_system_webauthn.ini [hunk 1]
```diff
@@ -22,7 +22,7 @@ PLG_SYSTEM_WEBAUTHN_ERR_NO_BROWSER_SUPPORT="Sorry, your browser does not support
 PLG_SYSTEM_WEBAUTHN_ERR_NO_STORED_CREDENTIAL="Cannot find the stored credentials for your login authenticator."
 PLG_SYSTEM_WEBAUTHN_ERR_NOT_DELETED="Could not remove the authenticator"
 PLG_SYSTEM_WEBAUTHN_ERR_USER_REMOVED="The user for this authenticator seems to no longer exist on this site."
-PLG_SYSTEM_WEBAUTHN_FIELD_DESC="Lets you manage passwordless login methods using the W3C Web Authentication standard. You need a supported browser and authenticator (eg Google Chrome or Firefox with a FIDO2 certified security key)."
+PLG_SYSTEM_WEBAUTHN_FIELD_DESC="Lets you manage passwordless login methods using the W3C Web Authentication standard. You need a supported browser and authenticator (eg Google Chrome or Firefox with a FIDO2 certified security key). <br /><br /><strong>Apple:</strong> Touch/Face ID <br /><strong>Windows:</strong> Hello (Fingerprint / Facial Recognition / PIN) <br /><strong>Android:</strong> Biometric screen lock"
```
Comment (brianteeman) [inline administrator/language/en-GB/plg_system_webauthn.ini]: Please change to just `<br>` and end the text with a period

Comment (tecpromotion) [reply administrator/language/en-GB/plg_system_webauthn.ini]: Perhaps also replace Apple as the manufacturer with the name of the operating system "MacOS/iOS/watchOS".
Should Linux be named?

Does Chrome OS now have enough market share and need to be mentioned?
A quick search on Google revealed that Chrome OS now has more market share than macOS.

**Comment by kitepascal** [comment]
Lets wait for @nikosdion s PR and some more tests.

Joomlas language override fields convert < br > to < br /> on save/reload, so I accepted that.

Should we also link @ceford s detail-documentation directly from PLG_SYSTEM_WEBAUTHN_FIELD_DESC ?

**Comment by brianteeman** [comment]
> Joomlas language override fields convert < br > to < br /> on save/reload, so I accepted that.


That's a bug which hopefully will get fixed. Please make the requested chanage

**Review by kitepascal — COMMENTED** [review]
image

**Comment by tecpromotion** [comment]
There are unnecessary spaces before and after br. Please delete.

Furthermore, I see no need for the last sentence and the link here.

**Comment by kitepascal** [comment]
I got used to add EOL spaces for anti-glued sentences if somehow line breaks get lost (or just take copy/paste w/o line breaks).

Unlikely here, so I will remove them (cannot hurt anyway).

For the "read more" link, last sentence - I think it is good and very, very useful to have them in scope, like "read more" in frontend. Its just following the intro/short form.

The top right help button is not really in scope and that last sentence with link does not hurt anything/anybody - rather the opposite.
The translations (+ possibly link(s)) and also the help button contents have to be updated anyway, if core Strings get improved for usability.

Maybe theres another 3rd party opinion on that - updated PR will follow.

File: administrator/language/en-GB/plg_system_webauthn.ini [hunk 2]
```diff
@@ -22,7 +22,7 @@ PLG_SYSTEM_WEBAUTHN_ERR_NO_BROWSER_SUPPORT="Sorry, your browser does not support
...
 PLG_SYSTEM_WEBAUTHN_ERR_NOT_DELETED="Could not remove the authenticator"
 PLG_SYSTEM_WEBAUTHN_ERR_USER_REMOVED="The user for this authenticator seems to no longer exist on this site."
-PLG_SYSTEM_WEBAUTHN_FIELD_DESC="Lets you manage passwordless login methods using the W3C Web Authentication standard. You need a supported browser and authenticator (eg Google Chrome or Firefox with a FIDO2 certified security key)."
+PLG_SYSTEM_WEBAUTHN_FIELD_DESC="Lets you manage passwordless login methods using the W3C Web Authentication standard. You need a supported browser and authenticator (eg Google Chrome or Firefox with a FIDO2 certified security key).<br><br><strong>MacOS/iOS/watchOS:</strong> Touch/Face ID.<br><strong>Windows:</strong> Hello (Fingerprint / Facial Recognition / PIN).<br><strong>Android:</strong> Biometric screen lock.<br><br>You can read about more possibilities and get detailed explanations in our Wiki:<br><a href="https://docs.joomla.org/Special:MyLanguage/WebAuthn_Passwordless_Login" title="How to use WebAuthn Passwordless Login" target="_blank">https://docs.joomla.org/WebAuthn_Passwordless_Login</a>."
```
Comment (tecpromotion) [inline administrator/language/en-GB/plg_system_webauthn.ini]:
```suggestion
PLG_SYSTEM_WEBAUTHN_FIELD_DESC="Lets you manage passwordless login methods using the W3C Web Authentication standard. You need a supported browser and authenticator (eg Google Chrome or Firefox with a FIDO2 certified security key).<br><br><strong>MacOS/iOS/watchOS:</strong> Touch/Face ID.<br><strong>Windows:</strong> Hello (Fingerprint / Facial Recognition / PIN).<br><strong>Android:</strong> Biometric screen lock.<br><br>You can find more details in the <a href='https://docs.joomla.org/Special:MyLanguage/WebAuthn_Passwordless_Login' target='_blank' rel='noopener noreferrer'>WebAuthn Passwordless Login documentation</a>."
```

File: administrator/language/en-GB/plg_system_webauthn.sys.ini [hunk 3]
```diff
@@ -4,4 +4,4 @@
 ; Note : All ini files need to be saved as UTF-8
 
 PLG_SYSTEM_WEBAUTHN="System - WebAuthn Passwordless Login"
-PLG_SYSTEM_WEBAUTHN_DESCRIPTION="Enables passwordless authentication using the W3C Web Authentication (WebAuthn) API. Please note that the WebAuthn tab in the user profile editor and the WebAuthn login buttons will only be displayed if the user is accessing the site over HTTPS. Furthermore, registering WebAuthn authenticators and using them to log into your site will only work when your site is using a valid certificate, signed by a Certificate Authority the user's browser trusts."
+PLG_SYSTEM_WEBAUTHN_DESCRIPTION="Enables passwordless authentication using Apple Touch/Face ID, Windows Hello, biometric screen lock on Android or other WebAuthn-compatible devices. Please note that the W3C WebAuthn tab in the user profile editor will only be displayed if the user is accessing the site over HTTPS. Furthermore, registering WebAuthn authenticators and using them to log into your site will only work when your site is using a valid certificate, signed by a Certificate Authority the user's browser trusts."
```
Comment (tecpromotion) [inline administrator/language/en-GB/plg_system_webauthn.sys.ini]:
```suggestion
PLG_SYSTEM_WEBAUTHN_DESCRIPTION="Enables passwordless authentication using the W3C Web Authentication (WebAuthn) API. Please note that the WebAuthn tab in the user profile editor and the WebAuthn login buttons will only be displayed if the user is accessing the site over HTTPS. Furthermore, registering WebAuthn authenticators and using them to log into your site will only work when your site is using a valid certificate, signed by a Certificate Authority the user's browser trusts."
```

**Review by tecpromotion — CHANGES_REQUESTED** [review]

**Comment by kitepascal** [comment]
Thx @tecpromotion - to change only the one string makes sense to save the translators crowd from unnecessary work (y)

File: administrator/language/en-GB/plg_system_webauthn.ini [hunk 4]
```diff
@@ -22,7 +22,7 @@ PLG_SYSTEM_WEBAUTHN_ERR_NO_BROWSER_SUPPORT="Sorry, your browser does not support
...
 PLG_SYSTEM_WEBAUTHN_ERR_NOT_DELETED="Could not remove the authenticator"
 PLG_SYSTEM_WEBAUTHN_ERR_USER_REMOVED="The user for this authenticator seems to no longer exist on this site."
-PLG_SYSTEM_WEBAUTHN_FIELD_DESC="Lets you manage passwordless login methods using the W3C Web Authentication standard. You need a supported browser and authenticator (eg Google Chrome or Firefox with a FIDO2 certified security key)."
+PLG_SYSTEM_WEBAUTHN_FIELD_DESC="Lets you manage passwordless login methods using the W3C Web Authentication standard. You need a supported browser and authenticator (eg Google Chrome or Firefox with a FIDO2 certified security key).<br><br><strong>MacOS/iOS/watchOS:</strong> Touch/Face ID.<br><strong>Windows:</strong> Hello (Fingerprint / Facial Recognition / PIN).<br><strong>Android:</strong> Biometric screen lock.<br><br>You can read about more possibilities and get detailed explanations in our Wiki:<br><a href="https://docs.joomla.org/Special:MyLanguage/WebAuthn_Passwordless_Login" title="How to use WebAuthn Passwordless Login" target="_blank">https://docs.joomla.org/Special:MyLanguage/WebAuthn_Passwordless_Login</a>."
```
Comment (Quy) [inline administrator/language/en-GB/plg_system_webauthn.ini:25]:
```suggestion
PLG_SYSTEM_WEBAUTHN_FIELD_DESC="Lets you manage passwordless login methods using the W3C Web Authentication standard. You need a supported browser and authenticator (eg Google Chrome or Firefox with a FIDO2 certified security key).<br><br><strong>MacOS/iOS/watchOS:</strong> Touch/Face ID.<br><strong>Windows:</strong> Hello (Fingerprint / Facial Recognition / PIN).<br><strong>Android:</strong> Biometric screen lock.<br><br>You can read about more possibilities and get detailed explanations in our Wiki:<br><a href=\"https://docs.joomla.org/Special:MyLanguage/WebAuthn_Passwordless_Login\" title=\"How to use WebAuthn Passwordless Login\" target=\"_blank\"  rel=\"noopener noreferrer\">https://docs.joomla.org/Special:MyLanguage/WebAuthn_Passwordless_Login</a>."
```

**Review by tecpromotion — APPROVED** [review]
thanks @kitepascal

**Comment by tecpromotion** [comment]
I have tested this item :white_check_mark: successfully on a4a2140fca76d957e5fffdf76aad931c29136dec<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37203">issues.joomla.org/tracker/joomla-cms/37203</a>.</sub>

**Comment by RickR2H** [comment]
I have tested this item :white_check_mark: successfully on a4a2140fca76d957e5fffdf76aad931c29136dec<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37203">issues.joomla.org/tracker/joomla-cms/37203</a>.</sub>

**Comment by RickR2H** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37203">issues.joomla.org/tracker/joomla-cms/37203</a>.</sub>

**Comment by bembelimen** [comment]
Thx

**Pull request merged**